web/beacon-app/src/features/topics/components/TopicQueryResult/MimeTypeResult/JSONResult.tsx

```diff
@@ -0,0 +1,19 @@
+import ReactJson from 'react-json-view';
+
+type JSONResultProps = {
+  data: any;
+};
+
+const JSONResult = ({ data }: JSONResultProps) => {
+  const formattedJson = JSON.parse(data);
+  return (
+    <ReactJson
+      src={formattedJson}
+      theme="summerfruit"
```

Comment:
Would it be possible to opt for the `colors` theme? This would offer the benefit of having a background color in black, aligning well with the black background of the parent component. Such a choice would ensure a distinct and cohesive background color.